core/src/main/java/org/opensearch/sql/ast/dsl/AstDSL.java

```diff
@@ -288,6 +289,10 @@ public UnresolvedExpression highlight(UnresolvedExpression fieldName,
     return new HighlightFunction(fieldName, arguments);
   }
 
+  public UnresolvedExpression position(UnresolvedExpression left, UnresolvedExpression right) {
+    return new PositionFunction(left, right);
```

Comment:
Same comment as above:  can we used a better description than left and right?

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9